config/cms/docker_run_production.py

```diff
@@ -352,7 +352,7 @@
 
 # Event Tracking
 TRACKING_IGNORE_URL_PATTERNS = config(
-    "TRACKING_IGNORE_URL_PATTERNS", default=None, formatter=json.loads
+    "TRACKING_IGNORE_URL_PATTERNS", default=[r"^/event", r"^/login", r"^/heartbeat"], formatter=json.loads
```

Comment:
After thought: why not using `common.py` default value instead for maintainability?

```suggestion
    "TRACKING_IGNORE_URL_PATTERNS", default=TRACKING_IGNORE_URL_PATTERNS, formatter=json.loads
```